client/blocks/reader-recommended-sites/style.scss

```diff
@@ -44,19 +44,26 @@
 	@include breakpoint( '>960px' ) {
 		align-items: flex-end;
 		flex-direction: column;
-		margin-top: -40px;
 	}
 
 	> :last-child {
 		width: 100%;
+		height: 100%;
 	}
 
 	.reader-recommended-sites__recommended-site-dismiss {
-		width: 30px;
+		display: flex;
+		justify-content: flex-end;
+		position: absolute;
+		top: 16px;
+		right: 16px;
 
 		@include breakpoint( '>960px' ) {
 			display: flex;
 			justify-content: flex-end;
+			position: absolute;
```

Comment:
I think that this and the `flex` wrapped in the breakpoint can be removed as they are already above. :)

Reply:
Ooh thanks for catching that!